pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/EventDao.sql.stg

```diff
@@ -1,5 +1,85 @@
 group EventDao;
 
+/**
+ * Returns all event configurations for study/activity and event type
+ */
+eventConfigurationLookup() ::= <<
+SELECT
+     ec.event_configuration_id,
+     ett.event_trigger_type_code event_trigger_type,
+     eat.event_action_type_code event_action_type,
+     ec.post_delay_seconds,
+     ec.dispatch_to_housekeeping,
+     (SELECT
+         e.expression_text from expression e
+     WHERE e.expression_id = ec.precondition_expression_id) as precondition_expression,
+     (SELECT
+         e.expression_text from expression e
+     WHERE e.expression_id = ec.cancel_expression_id) as cancel_expression,
+     ec.max_occurrences_per_user,
+     md.gcp_topic,
+     aist.activity_instance_status_type_code as instance_status_type,
+     ast.study_activity_id as activity_status_trigger_study_activity_id,
+     wst.workflow_state_id,
+     wst.trigger_automatically,
+     dnet.dsm_notification_event_type_code as dsm_notification_event_type,
+     ann_act.message_template_id as message_template_id,
+     nt.notification_type_code as notification_type,
+     ns.service_code as notification_service_type,
+     unea.notification_template_id,
+     unea.linked_activity_id,
+     pea.pdf_document_configuration_id,
+     unpdf.generate_if_missing,
+     aica.study_activity_id as activity_instance_creation_study_activity_id,
+     cp_target.copy_target as copy_answer_target,
+     qsc.stable_id as copy_source_question_stable_id
+FROM
+     event_configuration ec
+JOIN event_trigger et ON ec.event_trigger_id = et.event_trigger_id
+JOIN event_trigger_type ett ON et.event_trigger_type_id = ett.event_trigger_type_id
+JOIN event_action ea ON ec.event_action_id = ea.event_action_id
+JOIN event_action_type eat ON ea.event_action_type_id = eat.event_action_type_id
+JOIN umbrella_study study on ec.umbrella_study_id = study.umbrella_study_id
+LEFT JOIN message_destination md on ea.message_destination_id = md.message_destination_id
+LEFT JOIN activity_status_trigger ast ON ast.activity_status_trigger_id = ec.event_trigger_id
+LEFT JOIN activity_instance_status_type aist ON ast.activity_instance_status_type_id = aist.activity_instance_status_type_id
+LEFT JOIN activity_instance ai ON ai.study_activity_id = ast.study_activity_id
```

Comment:
Wait, why are we joining over to `activity_instance`? If user has multiple instances for the same `ast.study_activity_id`, then the result set will have _multiple rows for the same event config_. I don't see any `RowReducer`s being used in `EventDao`, which means we'll end up with duplicate event configuration objects...

Looks like it's only needed in the `getActivityStatusEventConfigurations` query below, so can we move the `left join` to down there?